x-pack/legacy/plugins/epm/server/lib/field.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
@skh @jfsiii Should we create a directory `fields` like we have for `template` or have also template also outside? As far as I can see so far there are no real pure technical pros / cons on how to group things? (in golang for example it has to do if variables are accessible).

Reply:
I tend to make a subdirectory as soon as I have a second file that would go in there (apart from tests). Either way is fine -- if we decide for separate directories I'll also move the `ingest_pipelines` files into one.